src/grpc/infer_handler.h

```diff
@@ -669,6 +670,33 @@ class InferHandlerState {
       return received_notification_ ? ctx_->IsCancelled() : false;
     }
 
+    // Extracts headers from GRPC request and updates state
+    void ExtractStateFromHeaders(InferHandlerStateType* state)
+    {
+      const auto& metadata = state->context_->ctx_->client_metadata();
+      for (const auto& pair : metadata) {
+        auto& key = pair.first;
+        std::string param_key = std::string(key.begin(), key.end());
+        std::string grpc_strict_key = "grpc_strict";
```

Comment:
Can we name this parameter "triton_grpc_error"?

Reply:
Is it possible to move this value to constants.h?

Reply:
Unable to find a constants.h in server.
Could you help point me to it, maybe I am missing something or vscode is unable to find it.

Reply:
You are right, it looks like we only have it for core.